errors.go

```diff
@@ -83,9 +98,9 @@ func HandleExitCoder(err error) {
 	if exitErr, ok := err.(ExitCoder); ok {
 		if err.Error() != "" {
 			if _, ok := exitErr.(ErrorFormatter); ok {
-				fmt.Fprintf(ErrWriter, "%+v\n", err)
+				_, _ = fmt.Fprintf(ErrWriter, "%+v\n", err)
 			} else {
-				fmt.Fprintln(ErrWriter, err)
+				_, _ = fmt.Fprintln(ErrWriter, err)
```

Comment:
⭐ _(nice to have)_ Given that this PR is already quite large, I would prefer that small style changes like this one ^ be removed.